tests/BasicSearchTests.FunctionalTests.Core/Constants.cs

```diff
@@ -13,6 +13,8 @@ public static class Constants
         public const string TestPackageSummary = "";
         public const string TestPackageAuthor = "clayco";
         public const string TestPackageCopyright = "Copyright 2013";
-        public const string NonExistentSearchString = "RandomCharacters234asdfa433";
+        public const string NonExistentSearchString = "asadfasdfsadfwerfasdf23423rafdsf";
```

Comment:
What if I go and create a package with such ID?

Reply:
For this case I typically use a double hyphen or a double dot:
http://regexstorm.net/tester?p=%5e%5cw%2b%28%5b_.-%5d%5cw%2b%29*%24&i=newtonsoft--json&o=in
https://github.com/NuGet/NuGet.Client/blob/503981224facaff322f0a68c399861fea6369327/src/NuGet.Core/NuGet.Packaging/PackageCreation/Utility/PackageIdValidator.cs#L14

Reply:
then it would break the test. We've been using the previous search string `RandomCharachters...` for our regular search tests, which didn't return any results. However, same doesn't work with Azure search as it creates tokens like `Random`, `Charachters`... thereby returning some results. I can't think of any search string that can't be broken by creating a package with such a search query unless counting for special characters.

Reply:
Ah, just saw @joelverhagen's comment. Will look into it.

Reply:
@joelverhagen - that regex is for the ID though. using hyphens and dots won't work with Azure search anymore since those are stop words thereby creating tokens that could actually match the search term.

Reply:
had offline chat with @joelverhagen - decided to leave this as is since that would reduce the scope of the test to that of only packageIds